hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -112,11 +112,23 @@
         Management Center can't access the data if at least one member has the data access disabled.
         Default value is true.
 
+        You can also specify <trusted-interfaces> as child element.
+        It includes allowed IP addresses for Management Center connections.
+        More preciselly, the element configures addresses from which clients are allowed to run cluster
+        management tasks over the Hazelcast client protocol.
+        You can give an IP addresses range using the wildcards (*) or intervals in the IPv4 address
+        (e.g. 192.168.1.* or 192.168.1.100-110).
```

Comment:
Why we don't have sentence like `Default value is empty, meaning all clients are allowed to run cluster management tasks.` also here (like we have it for yaml full example).

Reply:
I like the `Default value is empty` part, but `all clients are allowed to run cluster management tasks.` part may be misleading. Cluster management tasks code is not included into Java or any other client we provide, the code sits in private MC repo.
It's rather `addresses from where MC is allowed connect.`

Reply:
What about the `"... all Management Center connections can execute management tasks"`?

The thing is, we have the open client binary protocol, which contains the messages: https://github.com/hazelcast/hazelcast-client-protocol/blob/master/protocol-definitions/MC.yaml

And MC client can connect to the cluster even without matching trusted interfaces. It just fails to execute the management tasks then.